lib/elixir_newbie_web/router.ex

```diff
@@ -19,6 +19,7 @@ defmodule ElixirNewbieWeb.Router do
     # get "/", PageController, :home
     live "/", HomeLive, :home
     live "/blog", BlogLive, :blog
+    live "/blog/:id", BlogShow
```

Comment:
While no strictly necessary, I think we should provide a live-action atom for the sake of consistency with the other pages.

That, or we should remove the live actions from the other routes.

Consistency for the sake of consistency is sometimes important because right now, the reader asks "why does this one not need the atom?" which may lead to wasted time on a question that has no answer.

Reply:
Otherwise this is exactly how to handle routing, you mentioned you wanted feedback and my feedback is it looks good!

Reply:
Yes! I did not manage to make it work with the atom and, after a bit of tinkering, I got it to a working state, even though it did not seem consistent with the other routes.

My concern with routing is in `blog_card.ex`: the hyperlink in the blog card is generated this way:

`<a href={static_url(ElixirNewbieWeb.Endpoint, "/blog/#{@blog.id}")}>`

I do not know if this follows best practices.